File: node.js/node-telegram-bot-api/bot.js
const TelegramBot = require('node-telegram-bot-api');

const bot = new TelegramBot('TOKEN', {
  polling: true,
  baseApiUrl: 'https://pich-dev.herokuapp.com',
});

bot.onText(/\/start/, (message) => {
  bot.sendMessage(message.chat.id, "I'm a bot, please talk to me!");
});

File: python/python-telegram-bot/bot.py
from telegram.ext import CommandHandler
from telegram.ext import CallbackContext
from telegram import Update
from telegram.ext import Updater

updater = Updater(token='TOKEN', use_context=True,
                  base_url="https://pich-dev.herokuapp.com/bot")

dispatcher = updater.dispatcher


def start(update: Update, context: CallbackContext):
    context.bot.send_message(
        chat_id=update.effective_chat.id, text="I'm a bot, please talk to me!")


start_handler = CommandHandler('start', start)
dispatcher.add_handler(start_handler)

updater.start_polling()
